browser/net/brave_ad_block_tp_network_delegate_helper.cc

```diff
@@ -180,7 +180,7 @@ EngineFlags ShouldBlockRequestOnTaskRunner(
 
   SCOPED_UMA_HISTOGRAM_TIMER("Brave.Adblock.ShouldBlockRequest");
   g_brave_browser_process->ad_block_service()->ShouldStartRequest(
-      url_to_check, ctx->resource_type, source_host,
+      url_to_check, ctx->resource_type, source_host, ctx->aggressive_blocking,
```

Comment:
I gather that at some point we can craft a struct with needed parameters, to avoid changing all of the signatures